src/cmd/build.js

```diff
@@ -1,30 +1,31 @@
 /* @flow */
 import path from 'path';
-import minimatch from 'minimatch';
 import {createWriteStream} from 'fs';
-import streamToPromise from 'stream-to-promise';
+
+import minimatch from 'minimatch';
 import {fs} from 'mz';
+import streamToPromise from 'stream-to-promise';
 import parseJSON from 'parse-json';
 
 import defaultSourceWatcher from '../watcher';
 import {zipDir} from '../util/zip-dir';
 import getValidatedManifest, {getManifestId} from '../util/manifest';
 import {prepareArtifactsDir} from '../util/artifacts';
 import {createLogger} from '../util/logger';
-import {UsageError} from '../errors';
+import {UsageError} from '../errors'; //eslint-disable-line import/order
```

Comment:
Oh, so even though the comment disables the imports for types, it doesn't do it for a normal import and hence ESLint complains? I thought disabling ESLint for the type imports should be enough to deal with this. 
However, if we can club module and Flow imports, this won't be an issue at all. I'll go ahead and make changes to all the files in question.